test/Auth/authModule.js

```diff
@@ -0,0 +1,45 @@
+import test from 'ava';
+import { actionTest, reducerTest } from 'redux-ava';
+
+import {
+  changeUsername,
+  loginRequest,
+  reducer,
+} from '../../src/Auth/authModule';
+
+const username = 'ALi';
```

Comment:
Doesn't seems to be a good idea to have developer name in the code. Alice would just work as good and more neutral